lib/percona_migrator/command.rb

```diff
@@ -89,5 +93,10 @@ def log_started
     def log_finished
       logger.write("\n")
     end
+
+    def log_deprecations
+      logger.write("\n")
+      logger.write("[DEPRECATION] This gem has been renamed to Departure and will no longer be supported. Please switch to Departure as soon as possible.")
```

Comment:
I'd add a link to some page explaining a bit how to do that and the reason behind the rename. Maybe just a wiki page is enough.